apis/field_error_test.go

```diff
@@ -101,7 +101,18 @@ Body.`,
 		err:      ErrInvalidValue("foo", "bar"),
 		prefixes: [][]string{{"baz"}},
 		want:     `invalid value "foo": baz.bar`,
-	}}
+	}, {
+		name:     "mutually exclusive fields",
+		err:      ErrMutuallyExclusiveFields("foo", "bar"),
+		prefixes: [][]string{{CurrentField}},
```

Comment:
LOL, `CurrentField` is not meant to be used like this, just as a starting point.